### ISSUE TITLE
Investigate if we can make single-button shortcuts accessible by default

### ISSUE THREAD

**Issue by robintown** [vector-im/element-call#757]
…and then remove the setting if we succeed. A good starting point would be to look at how YouTube uses space as a shortcut without compromising accessibility. This should be timeboxed, and if we can't solve it then we should just keep the setting.
